mysensors/__init__.py

```diff
@@ -97,6 +97,65 @@ def add_sensor(self, sensorid=None):
             self.sensors[sensorid] = Sensor(sensorid)
         return sensorid if sensorid in self.sensors else None
 
+    def create_message_to_set_sensor_value(
+        self, sensor, child_id, value_type, value, **kwargs
+    ):
+        """Create a message to set specified sensor child value."""
+        default_ack = int(self.use_ack_when_set_values)
+
+        msg_type = kwargs.get("msg_type", self.const.MessageType.set)
+        ack = kwargs.get("ack", default_ack)
+
+        try:
+            value_type = int(value_type)
+        except ValueError:
+            _LOGGER.error(
```

Comment:
If we fail to create a message here it's also a developer mistake. We should raise an error.